modules/ob-usage-of-git-secret.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assembly:
+//
+// builds/authenticating-build-images.adoc
+
+:_content-type: CONCEPT
+[id="ob-usage-of-git-secret_{context}"]
+= Usage of Git secret
+
+After creating a secret in the relevant `OpenShift-Builds` namespace, you can reference it in your `Build` Custom Resource (CR). You can configure a Git secret with both types of authentication.
+
+The following example shows the usage of a Git secret with SSH authentication type:
+
+[source, yaml]
```

Comment:
This seems to be fine in the preview build, but I am not sure the portal transform will like it.
```suggestion
[source,yaml]
```